tools/documentation/.eleventy.js

```diff
@@ -71,7 +71,10 @@ module.exports = (config) => {
     return 'neutral';
   });
   config.addFilter('log', value => { console.log(value) });
-
+  config.addFilter('modifyImageSrc', function(value) {
```

Comment:
I think instead of adding an 11ty filter, we should update the metadata *.yml files to reflect the URL change. I did a quick search for `src="img` in `components/*/metadata` and found about 79 examples. @pfulton Do you think that makes sense?

Reply:
I was thinking the same but I didn't want to touch the existing work. But yea changing it in the existing files will be more relevant as this will take some amount of memory to carry out the task every time.

Reply:
@pfulton Please let us know which one of the above fixes do you want me to add

Reply:
Yeah, let's go the route of updating the Yaml files!

Reply:
@castastrophe  This is fixed now